docs/src/1_Mechanics/02_GMGuide.md

```diff
@@ -780,10 +778,10 @@ quality of that weapon determines how much damage.
 
 | Damage | Descriptor | Average Cost |
 | ------ | ---------- | ------------ |
-| 1      | Standard   | 5 gp         |
-| 2      | Improved   | 500 gp       |
-| 3      | Heroic     | 2,500 gp     |
-| 4      | Legendary  | 5,000 gp     |
+| 1      | Standard   | 15 sp        |
+| 2      | Improved   | 25 gp        |
+| 3      | Heroic     | 25 pp        |
```

Comment:
Will platinum `pp` be confused with Power Point `PP`? If yes, we could either invent a word here (akin to but not exactly etherium) or go with another substance (e.g., quartz pieces, qp)

Reply:
I agree, and when I wrote pp a bunch I saw it getting into similar territory but wanted to change everything to one uniform level higher for simplicity. 

I did a bit of research into some other areas to see other forms of currency or valued metals. Not finding many metals besides platinum that are traditionally associated with coins. However, I like the idea of a crystal and I think based on other systems (and generally, games) reliance on Diamonds it would be intuitive to make the higher level of currency diamond pieces `dp` and making those 10x the gp in this 'base' open source system's currency. Any (and most) systems and settings built on top of DofA will likely implement their own currency systems when it suits them based on genre or quantity need.

Reply:
Updated to Diamond Pieces `dp` in latest commit.